src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -302,8 +310,7 @@ class InvoiceForm extends Component {
                       </Col>
                       <Col data-test-col-accounting-code xs={3}>
                         <FieldSelection
-                          dataOptions={[]}
-                          disabled
+                          dataOptions={getCodeOptions(accountCodes)}
```

Comment:
Basing on the last comment from Dennis, labels of options should look like `acc number (acc code)`, please check that https://issues.folio.org/browse/UINV-36?focusedCommentId=56485&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-56485

Reply:
In this case, we'll have some problems when a vendor has acc number (as a required field) but hasn't got any acc code -> the option will look like ` acNumber123()`. So if a user selects this option we'll send to back-end empty string

Reply:
this field isn't required, I believe this should be fine